app/models/user.rb

```diff
@@ -0,0 +1,24 @@
+class User < ActiveRecord::Base
+  validates :email, :name, :uid, :provider, presence: true
```

Comment:
You may know this, but you can also use `validates_presence_of`instead of `validates` and `presence: true`

I think the former is more readable, when you are just doing these really simple validations. Just showing an alternate way; your way totally works :)